session/storage_memory.go

```diff
@@ -87,12 +91,34 @@ func (storage *StorageMemory) Find(id ID) (Session, bool) {
 	return Session{}, false
 }
 
+// ConsumeDataTransferedEvent consumes the data transfer event
+func (storage *StorageMemory) ConsumeDataTransferedEvent(e event.DataTransferEventPayload) {
```

Comment:
Can we separate event logic from storage itself like we do for all `Consume..` events

Reply:
???